dashboard/app/models/user_profile.rb

```diff
@@ -0,0 +1,93 @@
+# == Schema Information
+#
+# Table name: user_profiles
+#
+#  id             :integer          not null, primary key
+#  user_id        :integer          not null
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#  updated_by     :integer
+#  other_user_ids :string(255)
+#  other_emails   :string(255)
+#  course         :string(255)
+#  pd             :string(255)
+#  pd_manual      :string(255)
+#  properties     :text(65535)
+#
+# Indexes
+#
+#  index_user_profiles_on_user_id  (user_id)
+#
+
+class UserProfile < ActiveRecord::Base
+  belongs_to :user
+
+  include SerializedProperties
+  serialized_attrs %w(facilitator nmsi teals)
+
+  YEARS = [
+    YEAR_NONE = 'none'.freeze,
+    YEAR_2013_2014 = '2013-2014'.freeze,
+    YEAR_2014_2015 = '2014-2015'.freeze,
+    YEAR_2015_2016 = '2015-2016'.freeze,
+    YEAR_2016_2017 = '2016-2017'.freeze,
+  ].freeze
+  validates_inclusion_of :pd, in: YEARS, allow_nil: true
```

Comment:
The intent is for `nil` to represent "not set", with `NONE` representing "not PDed".